c2corg_ui/static/partials/cards/outings.html

```diff
@@ -16,11 +16,26 @@
         <span class="value-title glyphicon glyphicon-sort-by-attributes rotate-180"></span>
         <span class="value">{{::doc['elevation_max']}}&nbsp;m</span>
       </span>
+
+      <span class="location" ng-if="::doc['areas']">
+        <span class="glyphicon glyphicon-map-marker"></span>
+        <span class="value">{{::cardCtrl.showArea(doc['areas'])}}</span>
```

Comment:
This part is not correct (ditto for other card types).
The icon will be shown when the document has associated areas. But ``showArea`` will return something only for the feed items (see ``isTopoguide`` check).
Which means that the icon will appear in all cards even though no area is to be displayed. For instance in the outing advanced search page:

<img width="212" alt="capture d ecran 2016-12-12 a 15 51 50" src="https://cloud.githubusercontent.com/assets/1192331/21106027/c05d6092-c08c-11e6-914b-3f120f86fb47.png">